src/components/Swap/Input.tsx

```diff
@@ -150,7 +150,7 @@ export function InputWrapper({
           <Row>
             <USDC isLoading={isRouteLoading}>
               {usdc && `${formatCurrencyAmount({ amount: usdc, isUsdPrice: true })}`}
-              {impact && <ThemedText.Body2 color={impact.warning}>({impact.toString()})</ThemedText.Body2>}
+              {impact && <ThemedText.Body2 color={impact.warning}> ({impact.toString()})</ThemedText.Body2>}
```

Comment:
make this an explicit space {" "}

Reply:
I tried to - the linter refuses to let me.